core/src/com/unciv/ui/cityscreen/CitizenManagementTable.kt

```diff
@@ -0,0 +1,73 @@
+package com.unciv.ui.cityscreen
+
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.scenes.scene2d.Touchable
+import com.badlogic.gdx.scenes.scene2d.ui.Table
+import com.unciv.logic.city.CityFocus
+import com.unciv.models.stats.Stat
+import com.unciv.models.translations.tr
+import com.unciv.ui.images.ImageGetter
+import com.unciv.ui.utils.*
+
+class CitizenManagementTable(val cityScreen: CityScreen): Table() {
+    private val innerTable = Table()
+    val city = cityScreen.city
+
+    init {
+        innerTable.background = ImageGetter.getBackground(ImageGetter.getBlue().darken(0.5f))
+        add(innerTable).pad(2f).fill()
+        background = ImageGetter.getBackground(Color.WHITE)
+    }
+
+    fun update(visible: Boolean = false) {
+        innerTable.clear()
+
+        if (!visible) {
+            isVisible = false
+            return
+        }
+        isVisible = true
+
+        val focusLabel = "Default Focus".tr().toLabel().addBorder(5f, Color.CLEAR)
+        val defaultCell = Table()
+        defaultCell.add(focusLabel)
+        defaultCell.touchable = Touchable.enabled
+        defaultCell.onClick { city.cityAIFocus = CityFocus.NoFocus; city.reassignPopulation(); cityScreen.update() }
+        if (city.cityAIFocus == CityFocus.NoFocus)
+            defaultCell.background = ImageGetter.getBackground(BaseScreen.skin.get("selection", Color::class.java))
+        else
+            defaultCell.background = ImageGetter.getBackground(BaseScreen.skin.get("color", Color::class.java))
+        innerTable.add(defaultCell).growX().pad(3f).row()
+        for ((stat, _) in city.cityStats.currentCityStats) {
+            if (stat == Stat.Happiness) continue
+            if (stat == Stat.Faith && !city.civInfo.gameInfo.isReligionEnabled()) continue
+            val label = "${stat.name} Focus".tr().toLabel().addBorder(5f, Color.CLEAR)
```

Comment:
Double `tr()` (toLabel does one) and for the complex governors maybe another property on the enum.